compiler/rustc_session/src/options.rs

```diff
@@ -1585,7 +1585,7 @@ options! {
         "choose the TLS model to use (`rustc --print tls-models` for details)"),
     trace_macros: bool = (false, parse_bool, [UNTRACKED],
         "for every macro invocation, print its name and arguments (default: no)"),
-    track_diagnostics: bool = (false, parse_bool, [UNTRACKED],
+    track_diagnostics: bool = (true, parse_bool, [UNTRACKED],
```

Comment:
this doesn't look right? shouldn't it be off by default?

Reply:
Yes, I had this on while I was checking for any missing track_caller and accidentally committed it 😅